packages/lime-proto-bmx7/src/bmx7.lua

```diff
@@ -37,7 +39,7 @@ function bmx7.configure(args)
 	-- Enable JSON plugin to get bmx7 information in json format
 	uci:set(bmx7.f, "json", "plugin")
 	uci:set(bmx7.f, "json", "plugin", "bmx7_json.so")
-	
+
 	-- Enable SMS plugin to enable sharing of small files
```

Comment:
@axn does the SMS plugin add considerable overhead when unused? Maybe this should be configurable but default to true.

Reply:
AfAIK SMS plugin does nothing if it's not used.